packages/parser/src/parser.ts

```diff
@@ -220,7 +221,7 @@ export function createParser(options: ParserOptions) {
 
     const matchFn = memo((fnName: string) => {
       if (recipes.has(fnName) || patterns.has(fnName)) return true
-      if (fnName === cvaAlias || fnName === cssAlias || isFactory(fnName)) return true
+      if (fnName === cvaAlias || fnName === cssAlias || isRawFn(fnName) || isFactory(fnName)) return true
```

Comment:
This logic seems error-prune. It seems to match anything that ends with `.raw` and could be invalid sometimes.

For example:
```js
const x = String.raw``
```